packages/component-library/src/CardDetail/CardDetailPage.js

```diff
@@ -28,7 +27,16 @@ const CardDetailPage = ({ params }) => {
 };
 
 CardDetailPage.propTypes = {
-  params: PropTypes.string
+  params: PropTypes.shape({ slug: PropTypes.string.isRequired }),
+  CardRegistry: PropTypes.shape({
+    entries: PropTypes.arrayOf(
+      PropTypes.shape({
+        component: PropTypes.func.isRequired,
+        project: PropTypes.string.isRequired,
+        slug: PropTypes.string.isRequired
+      })
+    )
+  })
```

Comment:
Since we're doing `.isRequired`, that can be added at the end of the `CardRegistry` prop also.